### ISSUE TITLE
AP, PQ, SQ, RQ=0 in training

### ISSUE THREAD

**Issue by EnnioEvo** [facebookresearch/Mask2Former#61]
Thank you for this amazing work,

I am trying to replicate the results on the coco dataset so I am training running

`python3 train_net.py --config-file configs/coco/panoptic-segmentation/maskformer2_R50_bs16_50ep.yaml --num-gpus 1 SOLVER.IMS_PER_BATCH 1 SOLVER.BASE_LR 0.000001`

and what I get, even after 200k iterations is
![screen1](https://user-images.githubusercontent.com/26523410/157229934-4e6087e0-98e8-4bf0-8657-5e1889ba55e3.png)

same 0 for PQ, SQ and RQ.
If I run the inference with my trained weigths I just get the same image with no labels. (inference with pretrained weigths instead works fine).
What can be? Maybe the learning rate too low? But higher LR gives me "invalid matrix entries" after a while

**Comment by bowenc0221** [comment]
The learning rate is probably too small.
